packages/components/src/ui/item-group/context.ts

```diff
@@ -4,6 +4,7 @@
 import { createContext, useContext } from '@wordpress/element';
 
 export const ItemGroupContext = createContext( { size: 'medium' } as {
+	spacedAround: boolean;
```

Comment:
Shall we give this a default value in the context, just for consistency?

Reply:
We could! I wasn't sure how to go about this, since it's kind of inferred from `bordered` / `separated` 🤔